Online Learning Management/Presentation/Controllers/CoursesController.cs

```diff
@@ -79,20 +93,5 @@ public async Task<IActionResult> Delete(Guid courseId)
             return Ok(new { message = "Course was deleted succesfully" });
         }
 
-        [HttpGet("/instructors/{IdInstructor}/courses")]
-        public async Task<IActionResult> GetCoursesByIdInstructor(int IdInstructor)
```

Comment:
That endpoint that you deleted "GetCourse ByIdInstructor", I think it is necessary to just have it, or how are you bringing the courses from an instructor?

Reply:
I talked to Nico and we decided that, this endpoint will be in the Instructors controller

Reply:
Oki 👍